bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Recaptcha.java

```diff
@@ -17,19 +17,23 @@
 
 import org.osgi.annotation.versioning.ConsumerType;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 /**
- * Defines the form {@code reCaptcha} Sling Model used for the {@code /apps/core/fd/components/form/captcha/v1/captcha}
+ * Defines the form {@code reCaptcha} Sling Model used for the {@code /apps/core/fd/components/form/recaptcha/v1/recaptcha}
  * component.
  *
  * @since com.adobe.cq.forms.core.components.models.form 2.0.0
  */
 @ConsumerType
 public interface Recaptcha extends Field {
```

Comment:
Can we create an AbstractCaptcha class so that other other captcha implementation when created in future can extend it ?